src/components/sitepages/SiteTargets.vue

```diff
@@ -748,6 +712,46 @@ export default {
 
   methods: {
 
+    async intializeAladinView () {
+      this.$loadScript('https://aladin.u-strasbg.fr/AladinLite/api/v2/latest/aladin.min.js')
+        .then(async () => {
+        // Default: load aladin on m17, but use coords in mount fields if possible.
+          let target = 'M17'
+          if (parseFloat(this.mount_ra) && parseFloat(this.mount_dec)) {
+            target = `${15 * this.mount_ra} ${this.mount_dec}`
+          }
+
+          // Initialize Aladin
+          this.aladin = A.aladin('#aladin-lite-div', {
+            survey: 'P/DSS2/color',
+            fov: this.camera_size_degrees,
+            target,
+            cooFrame: 'ICRSd',
+            showFullscreenControl: false,
+            showGotoControl: false,
+            showSimbadPointerControl: true
+          })
+          // cheap way to sync the skymap and mount coordinates to the aladin view.
+          setTimeout(this.set_coordinates_from_aladin, 1000)
+
+          // The following code creates an overlay element that hides aladin from mouseevents.
+          // This is mainly to let the user scroll the sidebar without aladin zooming instead.
+          // If the user clicks on aladin, then normal mouse behavior returns.
+          const scroll_hide_overlay = document.createElement('div')
```

Comment:
Here's my suggestion for how to handle this in a more maintainable and readable way:

Under data, you can return the following

```
 aladin: null,
    isOverlayVisible: true,
    overlayStyle: {
      height: '100%',
      width: '100%',
      position: 'relative',
      zIndex: 4,
```

then have a handler method, maybe called `handleOverlayClick` and here you change `isOverlayVisible` to `false`. You then can write a dynamic element in the template, something like: 

```
<div id="aladin-lite-div">
  <div 
    v-if="isOverlayVisible" 
    :style="overlayStyle" 
    @click="handleOverlayClick">
  </div>
</div>
```

I don't entirely understand the intended behavior, so adapt it as needed. But that's my main suggestion

Reply:
I didn't really understand it either as it's Tim's code, but I can look into your suggestions if you think it needs rewriting

Reply:
why does this appear as new code?

Reply:
when you move code in git it shows a deletion at one area and then an addition in another. So bringing this code into its own function causes the lines from 681-717 to appear as deleted and 715-754 to appear as inserted. When in fact it was separating it into its own method 'intializeAladinView()' at line 618. Green = new code is not always the case in Github.

Reply:
If it was like that, then I think it's fine. I didn't know this was a github behavior. The more you know! Looks good to me, then!